docs/DeveloperGuide.md

```diff
@@ -270,64 +271,83 @@ shows the updated Sequence diagram for the executing of our `delete` command.
-`setCellValueFactory` which allows us to iterate through the list of `Residents` and obtain the value dynamically.
+Alternative 2: List
 
-In using the `setCellValueFactory` method, we also used the `PropertyValueFactory` class. The implementation of
-`PropertyValueFactory` has enabled us to easily obtain fields due to its *method matching*
-[functionality](https://docs.oracle.com/javase/8/javafx/api/javafx/scene/control/cell/PropertyValueFactory.html).
+Pros:
+* Ability to customize the layout of the data to be displayed
+* Ability for the user to more focused by viewing lesser amounts of information at a glance
 
-For example, by constructing `nameCol.setCellValueFactory(new PropertyValueFactory<Resident, String>("name")`,
-the "name" string is used as a reference to an assumed `Resident::getName` in `Resident.java`.
-As a consequence of this, for all fields in `Resident`, we have to implement and have implemented suitable methods to
-take advantage of this functionality. The only caveat to this is potentially the complexity of fields that a `Resident`
-could possess. But for our purposes, our fields are mainly represented as `String`, and there are no issues thus far.
+Cons:
+* Possible lack of information displayed at a glance
+* Less intuitive for handling large numbers of entries
 
-<br>
 
-##### Differences in Updating Data
+For the purposes of the user, who has to deal with large amounts of residential information, we opted the use of the table.
+
+
+#### Booking Information
+
+Similar to the display of resident information, the `VenueTabView` contains a `BookingTableView` which was also implemented
+via the `TableView` class of `JavaFX`. Here, each row corresponds to the `Day`, and each column corresponds to the `HourPeriod`.
+
+##### Design considerations
+
+Aspect: Display format
+
+Alternative 1 (current choice): Table
 
-Another difference between `PersonListPanel` and `ResidentTableView` is the behavior in propagating changes in the
-`Model` component to the `Ui` component. In `ResidentTableView` modifications to any fields of a `Resident` would not require explicit
-invocation of a method to update the Ui. This design was possible as `TableView` automatically adds an observer
-to the returned value from `setCellValueFactory`, as mentioned in the [section above](#obtaining-resident-fields).
-As a result, any updates to `ObservableList<Resident>` would be reflected immediately in all cells of the Table.
+Pros
+* Decently proficient in displaying bookings
+
+Cons
+* Difficulty in customizing the design i.e. colors and size
+
+Alternative 2: Grid
+
+The implementation of the grid would be via a 8x16 grid of JavaFX containers using `HBox` and `VBox`. We would have nested
+the `HBox` and `VBox` components to achieve a timetable like design to display the bookings.
+
+Pros
+* Decently proficient in displaying bookings
+* Easier to customize the design i.e. colors and size
+
+Cons
+* Difficulty and time-consuming to implement
```

Comment:
Minor grammatical error. "Difficult and time-consuming to implement".

Reply:
The cons for the other part is okay because of the sentence structure, though if you want, you could change to "Difficult to customize the design"